doc/radosgw/adminops.rst

```diff
@@ -275,6 +275,9 @@ Get User Info
 
 Get user information.
 
+Either a ``uid`` or ``access-key`` must be supplied as a request parameter. We recommend supplying uid.
```

Comment:
Just noticed that the `uid` at the end lacks backticks.

Reply:
I'll make a follow-up PR against /main that adds this (and probably some other odds and ends that I've been keeping track of).